languages/elixir/exercises/concept/rpg-character-sheet/.docs/instructions.md

```diff
@@ -0,0 +1,61 @@
+You and your friends love to play pen-and-paper role-playing games, but you noticed that it's difficult to get new people to join your team. They often struggle with character creation. They don't know where to start. To help new players out, you decided to write a small program that will guide them through the process.
```

Comment:
```suggestion
You and your friends love to play pen-and-paper role-playing games, but you noticed that it's difficult to get new people to join your group. They often struggle with character creation. They don't know where to start. To help new players out, you decided to write a small program that will guide them through the process.
```

Reply:
I think group is more appropriate than team here. (Speaking as a nerd)

Reply:
I'll take the lack of any other comments from you to mean that an IO exercise is allowed 😉